netlify.toml

```diff
@@ -11,3 +11,9 @@ HUGO_VERSION = "0.69.0"
 
 [context.branch-deploy.environment]
 HUGO_VERSION = "0.69.0"
+
+[[redirects]]
+from = "/support/GrandStGuild"
+to = "/support/"
+status = 302
+force = true
```

Comment:
Any reason this can't be done centrally, in `static/_redirects`?

Reply:
Maybe? I was not aware the _redirects was a thing.

Reply:
The netlify docs specify redirects in netlify.toml, so that's where I've been doing them. Is _redirects a hugo thing?

Reply:
https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file

It supports both, but up to this point we have only been using `_redirects`, so it would be good to keep everything in one place

Reply:
Done